src/js/common/pages/Politician/PoliticianDetailsPage.jsx

```diff
@@ -899,7 +900,7 @@ class PoliticianDetailsPage extends Component {
               </CandidateCampaignListMobile>
             )}
           </DetailsSectionMobile>
-          <DetailsSectionDesktopTablet className="u-show-desktop-tablet">
+          <DetailsSectionDesktopTablet style={uShowDesktopTablet()}>
```

Comment:
I'm approving this pull request, but before we spread this design pattern, do the uShowDesktopTablet and uShowMobile functions adjust responsively on desktop when you resize the window? Thank you in advance.